rpc/gnmi/gnmi-union_replace.md

```diff
@@ -0,0 +1,327 @@
+# union_replace gNMI method
```

Comment:
(nit) can you 80ch wrap this file please?

Reply:
Done